modules/dicom_archive/README.md

```diff
@@ -3,8 +3,8 @@
 ## Purpose
 
 The DICOM Archive module provides a front end to view the metadata
-about DICOMs which have been inserted into the LORIS database via
-the imaging pipeline.
+about DICOMs which have been processed by the LORIS database via
```

Comment:
@driusan I don't really like the "processed" term. The imaging pipeline does not "process" the images, it simply registers the DICOM information into the SQL database. No processing involved. Does that make sense?

Reply:
I agree, but I don't like the word "inserted" either because it appears in the dicom archive even if it wasn't inserted into the imaging browser when it fails the pipeline, which is what people interpret as "inserted into LORIS".

Do you (or anyone) have any other suggestions?

Reply:
What about something like:
"The DICOM Archive module provides a front end to view the metadata about DICOM studies and series that have been registered in the LORIS database via the first step of the imaging pipeline (using dicomTar.pl). The information displayed in this module includes all modalities of the DICOM studies (including both failed and successful acquisitions). Protocol checks and QC are performed later on and not represented in this module."

Reply:
I would refrain from talkibg about failed and successful acquisitions as the DICOM archive precedes these steps. 
I would say something like:’ the module displays metadata of the Uploaded/transferred scans’ DICOMs the way they are archived in the various tarchive_* tables by the pipeline insertion scripts, I.e. at the study, series, and file levels’

Reply:
What if I just left out the second last sentence in Cecile's proposal?

"The DICOM Archive module provides a front end to view the metadata about DICOM studies and series that have been registered in the LORIS database via the first step of the imaging pipeline (using dicomTar.pl).  Protocol checks and QC are performed later on and not represented in this module."

Reply:
Agree with Mouna, the failed and successful acquisitions might be interpreted in different ways.

Your suggestion about removing the second last sentence sounds good to me!

Reply:
sounds good to me too!!!

Reply:
I put that in, can you guys reread it and review it as a whole?